drakrun/drakrun/lib/networking.py

```diff
@@ -1,10 +1,30 @@
 import logging
 import os
+import re
 import signal
 import subprocess
 from typing import Optional
 
-log = logging.getLogger("drakrun")
+log = logging.getLogger(__name__)
+
+
+def find_default_interface() -> Optional[str]:
+    routes = (
+        subprocess.check_output(
```

Comment:
subprocesses eveyrywhere eh, it would be nice to have all subprocesses encapsulated nicely in one/two files.